### PR TITLE
[FEAT] add close icon to modals

### PR DESCRIPTION
**Pull request by JacobSobolev**
# Description

based on PR #578 I added close icons to all the models that didn't have

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
`yarn test`
image

### All the models with the close icon (close functionality has been tested):

image

image

image

image

image

image

image

image

image

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/components/Lore.tsx [hunk 1]
```diff
@@ -47,6 +48,11 @@ export const Lore: React.FC = () => {
       {showGreenBook && (
         <Modal centered show={showGreenBook} onHide={() => onCloseBook()}>
           <Panel>
+            <img
+              src={close}
+              className="h-6 top-4 right-4 absolute cursor-pointer"
+              onClick={() => onCloseBook()}
```
Comment (ahh1539) [inline src/features/game/components/Lore.tsx:54]: Purley a nitpick, so take it or leave it but the *best* way to handle function calls on event handlers in react is `onClick={onCloseBook}`

for reference: https://upmostly.com/tutorials/react-onclick-event-handling-with-examples

File: src/features/game/components/Lore.tsx [hunk 2]
```diff
@@ -77,6 +83,11 @@ export const Lore: React.FC = () => {
       {showYellowBook && (
         <Modal centered show={showYellowBook} onHide={() => onCloseBook()}>
           <Panel>
+            <img
+              src={close}
+              className="h-6 top-4 right-4 absolute cursor-pointer"
+              onClick={() => onCloseBook()}
```
Comment (ahh1539) [inline src/features/game/components/Lore.tsx:89]: onClick={onCloseBook}

File: src/features/game/components/Lore.tsx (hunk 1 above)
Comment (0xsha1man) [reply src/features/game/components/Lore.tsx:54]: > Purley a nitpick, so take it or leave it but the "best" way to handle function calls without parameters on event handlers in react is `onClick={onCloseBook}`
> 
> for reference: https://upmostly.com/tutorials/react-onclick-event-handling-with-examples

It's actually very good practice to do this. You're avoiding additional javascript event firing and calls/parsing of the dom. As the game gets more complex script executions like that can snowball and hit performance.

**Comment by tiffanydys** [comment]
Hi! I noticed that you added a kitchen button here:
image

Suggestion: might as well add it to the other goblin modals? What do you think

**Comment by newb23** [comment]
> Suggestion: might as well add it to the other goblin modals? What do you think

If he does not, I would love to add these!

**Comment by ahh1539** [comment]
> If he does not, I would love to add these!

You could do this for the axe

**Comment by newb23** [comment]
> You could do this for the axe modal -> crafting
I certainly can!

File: src/features/game/components/Lore.tsx (hunk 1 above)
Comment (JacobSobolev) [reply src/features/game/components/Lore.tsx:54]: @ahh1539 
sure I can change this according to this but I just reused the previous code that was already there.
so if we are fixing this we need to fix the code previously as well, but not sure this needs to be in this PR

**Comment by JacobSobolev** [comment]
> Hi! I noticed that you added a kitchen button here: image
> 
> Suggestion: might as well add it to the other goblin modals? What do you think

I did it for all of them, look at the other screenshots

**Comment by JacobSobolev** [comment]
> > Suggestion: might as well add it to the other goblin modals? What do you think
> 
> If he does not, I would love to add these!

yea, I will handle this, no worries :)

**Comment by JacobSobolev** [comment]
@newb23 @ahh1539 I have added the close icon to all models including the lumberjack.
the only one I didn't add is to the wishing well because it is unfinished and probably going to change.

**Review by adamhannigan — APPROVED** [review]
Nicely done! Much more consistent 💪

**Pull request merged**